### ISSUE TITLE
Mark WPA and source lints when printing to console

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#518]
I'm linting my EO program and getting some warnings in my console. For example:
```
[WARNING] org.eolang.io.tee-input:54 Object '.write' has arguments inconsistency (inconsistent-args)
```
It's not clear what kind of lint `inconsistent-args` is - individual or WPA. So I'm not able to suppress it programmatically without checking the source code of `lints` repository. 

It would be convenient to mark somehow WPA and source lints when printing them to console, for instance

```
[WARNING] org.eolang.io.tee-input:54 Object '.write' has arguments inconsistency (inconsistent-args, WPA)
```

**Comment by h1alexbel** [comment]
@maxonfjvipon the `inconsistent-args` lint is pure WPA lint, it just checks each XMIR individually too (to build a map of 
object usages and their argument counts). I thought about adding the information about with which usage reported usage clashes. For instance:

```text
[WARNING] org.eolang.io.tee-input:54 Object '.write' has arguments inconsistency (clashes with a:52, b:42, c:11) (inconsistent-args)
```

WDYT?

**Comment by maxonfjvipon** [comment]
@h1alexbel yes, its' WPA lint, but it's unclear when I see the warning in console with many other warnings. Imagine I want to suppress all the warnings I see in console and I see this: 
```[WARNING] org.eolang.io.tee-input:54 Object '.write' has arguments inconsistency (inconsistent-args)
```

What should I do:
```java
new Source(...).without("inconsistent-args")
// OR
new Program(...).without("inconsistent-args")
```
I need to go to source code where I can find that `inconsistent-args` is WPA. But If I would see something like:
```
[WARNING] org.eolang.io.tee-input:54 Object '.write' has arguments inconsistency (inconsistent-args)
```
I would definitely know that I need to do:
```java
new Program(...).without("inconsistent-args")
```

It's not about this particular lint but about the difference between these two types of lints in general. When we report them, I think we should provide the type of lint too

**Comment by h1alexbel** [comment]
@maxonfjvipon ah, got it. Indeed, let's adjust it as well. First, let me add clash information in the `inconsistent-args` lint, and then I will update `Source` and `WPA` lints with respective marker.

**Comment by 0crat** [comment]
@maxonfjvipon Great job reporting that new bug! As per our policy, you've earned **8** points for your contribution. Your keen eye helps us enhance our project's quality. Remember, if you spot anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) up to par, [don't hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. We value your input! Your current balance stands at [+124](https://www.eolang.org/zerocracy/objectionary-vitals.html) points. Keep up the excellent work! 🐛🔍

**Comment by h1alexbel** [comment]
@maxonfjvipon please try new version: [0.0.48](https://github.com/objectionary/lints/releases/tag/0.0.48). It should solve the issue

**Comment by VasilevNStas** [comment]
@yegor256 @volodya-lombrozo @maxonfjvipon, this issue became **moot on `master`**: the whole-program-analysis (WPA) layer — including all WPA lints like `inconsistent-args` — was completely removed in commit `235cda2d` (PR #845, Apr 2026). There is no WPA/Source distinction to print anymore, since only source-level lints remain.

I'd suggest closing this as resolved-by-refactoring.

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon I guess, this one was fixed. If it's still an issue, please, feel free to open a new one in the wpa repo:
https://github.com/objectionary/wpa
